docs/user-flows.md

```diff
@@ -1,262 +1,153 @@
 # User Flows in Lighthouse
 
-> Motivation: You want to run Lighthouse on your whole site, not just the landing page.
+Historically, Lighthouse has analyzed the cold pageload of a page. Starting in 2022 (Lighthouse v10), it can analyze and report on the entire page lifecycle via "user flows".
 
-> Is that checkout form after you add an item to your cart accessible?
+#### You might be interested in flows if…
 
-> What is the Cumulative Layout Shift of my SPA page transition?
+* … you want to run Lighthouse on your whole webapp, not just the landing page.
+* … you want to know if the checkout process is accessible.
+* … you want to know the Cumulative Layout Shift of my SPA page transition
 
-You want Lighthouse on a _flow_, not just a page load.
+In these cases, you want Lighthouse on a _flow_, not just a page load.
 
-This document describes how Lighthouse measures flows and offers recommendations on how to structure your own flow measurement with practical examples.
+## The three modes: Navigation, Timespan, Snapshot
 
-## Flow Building Blocks
+Lighthouse can now run in three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
 
-Flow measurement in Lighthouse is enabled by three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
+* **Navigation mode** analyzes a single page load. Prior to v10, all Lighthouse runs were essentially in this mode.
```

Comment:
Not sure if we should be using v10 as the time origin since the modes were released with v9.6.1